### ISSUE TITLE
query on baseline translation quality

### ISSUE THREAD

**Issue by wjbyrne** [ZurichNLP/understanding-mbr#1]
Hi, 
I'm working through your scripts (for which, thank you!) and would like to check that the Dan-Epo system that I produced is as expected.  I ran the scripts more or less as expected, although from the command line, without SLURM.   In computing BLEU score, I get:

cat evaluations/dan-epo/no_label_smoothing/test.beam.1.0.top.10.bleu 
BLEU+case.mixed+numrefs.1+smooth.exp+tok.13a+version.1.4.14 = 25.875 
61.0/32.1/20.0/12.7 (BP = 0.974 ratio = 0.975 hyp_len = 33965 ref_len = 34844)

Does that look correct?   In looking at Figure 9 of your appendix I expected a BLEU score closer to 30.0 or so.

Thanks,
Bill

**Comment by bricksdont** [comment]
Hi Bill

There are several test sets (which ones exactly depends on the settings in the main script). There should be at least `test` and `slice-test`. `test` is the Tatoeba test set (https://github.com/Helsinki-NLP/Tatoeba-Challenge).

But the Tatoeba test set can be considered out-of-domain.`slice-test` is a slice of the training data to ensure an in-domain test. On `slice-test` my BLEU score is roughly 30.0:

    cat evaluations/dan-epo/no_label_smoothing/slice-test.beam.1.0.top.10.bleu
    BLEU+case.mixed+numrefs.1+smooth.exp+tok.13a+version.1.4.14 = 31.625 61.1/37.5/26.0/18.9 (BP = 0.971 ratio = 0.972 hyp_len = 28937 ref_len = 29776)

and for reference:

    cat evaluations/dan-epo/no_label_smoothing/test.beam.1.0.top.10.bleu
    BLEU+case.mixed+numrefs.1+smooth.exp+tok.13a+version.1.4.14 = 25.477 59.8/31.1/19.3/12.2 (BP = 0.990 ratio = 0.990 hyp_len = 34496 ref_len = 34844)

Additional comments that may be helpful:

- File names mean: `[test set].[decoding algorithm].[length normalization alpha].top.[beam size].[metric]`
- Figure 9 in the appendix does not show any results with beam search, only MBR with different utility functions.

Thanks for testing our code by the way, and let me know if this is helpful

Mathias

**Comment by bricksdont** [comment]
P.S. Did you have to make any changes to make the scripts run without SLURM? I'd be interested to know since I never tested without.

**Comment by wjbyrne** [comment]
Thanks, Mathias.   That's all very helpful.  Much appreciated.    Btw, I did see that Fig 9 shows only MBR results, but I figured the beam search results should be somewhat similar.

Yes,  the non-SLURM scripts ran well , at least as far as I can see so far.  Installation was a little tricky.  I couldn't get pyenv to run on my host, and so I wasn't able to use your create_venv.sh script directly.    But I was able to install all the needed tools into my existing miniconda environment, and all the scripts ran fine,  apart from some harmless calls to non-existent python environments.    

Bill
